docs/admin/install/install.rst

```diff
@@ -152,13 +156,13 @@ With the key and configuration available in ``dom0``, you're ready to set up Sec
 
   .. code-block:: sh
 
-    rpm -Kv securedrop-workstation-dom0-config-<versionNumber>-1.fc32.noarch.rpm
+    rpm -Kv securedrop-workstation-dom0-config-<versionNumber>-1.fc37.noarch.rpm
 
   where ``<versionNumber>`` is the release version number you noted above. The command output should match the following text:
 
   .. code-block:: none
 
-    securedrop-workstation-dom0-config-<versionNumber>-1.fc32.noarch.rpm:
+    securedrop-workstation-dom0-config-<versionNumber>-1.fc37.noarch.rpm:
       Header V4 RSA/SHA512 Signature, key ID 7b22e6a3: OK
       Header SHA256 digest: OK
       Header SHA1 digest: OK
```

Comment:
I don't see the `  V4 RSA/SHA512 Signature, key ID 7b22e6a3: OK` line (right above MD5 digest) anymore when I run this in the Fedora 40-based VM.